packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -22,14 +22,12 @@ import type {RootTag} from 'react-reconciler/src/ReactRootTags';
 import * as Scheduler from 'scheduler/unstable_mock';
 import {REACT_FRAGMENT_TYPE, REACT_ELEMENT_TYPE} from 'shared/ReactSymbols';
 import {ConcurrentRoot, LegacyRoot} from 'react-reconciler/src/ReactRootTags';
```

Comment:
I wonder why these work. Should we move them too?

Reply:
Probably yeah